05_syntax_lab/01.py

```diff
@@ -1,5 +1,18 @@
-"""
-Write a program that reads 10 numbers from
-the user and prints the largest one
-"""
-
+"""
+Elad Agam
+01 - Get 10 numbers and return Max
+"""
+
+#init max
+print "Enter a Number"
+input = raw_input()
+Max = input
+
+for num in range(9):
+	print "Enter another Number"
+	input = raw_input()
+	if input > Max:
+		Max=input
+
+print "Max = {maximum}".format(maximum=Max)
+

```

Comment:
Hi Elad,
thanks for your first solution.
Your solution seems to work, but there can be a number of improvements. Please try them:
* Variable names are usually all lowercase, but ```max``` is also a keyword in Python. Could you use a different variable name?
* Could you eliminate the duplicate call to ```raw_input```?
* The call to ```format``` with the name is nice very nice!
* The "Enter another Number" now prints a newline before the user can enter the input. Could you get rid of that newlines?